### PR TITLE
add test for getJettonData getWalletData getMaxPunishment

### PR DESCRIPTION
**Pull request by MKdir98**
This PR is for https://github.com/HipoFinance/contract/issues/3
My telegram ID: [MK_dir98](https://t.me/MK_dir98)

### CODE REVIEW

File: tests/Getter.spec.ts
Comment (behrang) [inline tests/Getter.spec.ts:1]: Rename the file to 'Getters' so it matches the `describe` function.

File: tests/Getter.spec.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+        fees = await treasury.getFees()
+
+        await treasury.sendTopUp(deployer.getSender(), { value: fees.treasuryStorage })
+    })
+
+    it('should deploy treasury', async () => {
+    })
+
+    it('should getters work', async () => {
```
Comment (behrang) [inline tests/Getter.spec.ts]: Use separate test for each getter and improve names. For example, 'should return maxPunishment`, 'should return Jetton data', 'should return wallet data' etc.

Also, please add tests for other get methods like `get_treasury_state` and `get_wallet_state` and check all the returned values to be correct, i.e. match the value in the deploy function.

**Review by behrang — CHANGES_REQUESTED** [review]

**Comment by MKdir98** [comment]
I tried to fix them as much as I could. Please check it again and tell me if it has problem

**Comment by behrang** [comment]
Good job. Thanks.

**Pull request merged**